src/main/java/task/NewOpportunityPortal/cryp/EncryptDecrypt.java

```diff
@@ -0,0 +1,43 @@
+package task.NewOpportunityPortal.cryp;
+
+
+import lombok.RequiredArgsConstructor;
+import org.apache.commons.codec.binary.Base64;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import javax.crypto.BadPaddingException;
+import javax.crypto.Cipher;
+import javax.crypto.IllegalBlockSizeException;
+import javax.crypto.NoSuchPaddingException;
+import javax.crypto.spec.SecretKeySpec;
+import java.io.UnsupportedEncodingException;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+@Component
+public class EncryptDecrypt {
+
+    private static final String SECRET_KEY = "f789054kjl";
```

Comment:
Secret keys, passwords to database etc should be kept in the environment variable and you should retrieve all such values from there